packages/amplify_auth_cognito/lib/method_channel_auth_cognito.dart

```diff
@@ -254,6 +254,26 @@ class AmplifyAuthCognitoMethodChannel extends AmplifyAuthCognito {
     return res;
   }
 
+  @override
+  Future<SignInResult> signInWithWebUI({SignInWithWebUIRequest request}) async {
+   SignInResult res;
+    try {
+      final Map<String, dynamic> data =
+      await _channel.invokeMapMethod<String, dynamic>(
+        'signInWithWebUI',
+        <String, dynamic>{
+          'data': request != null ? request.serializeAsMap() : null,
+        },
+      );
+      return _formatSignInResponse(data, "signIn");
+    } on PlatformException catch(e) {
+      _throwError(e);
+    } on AuthError catch(e) {
+      throw(e);
+    }
+    return res;
+  }
```

Comment:
Do we need an API review of this? Native seems to have this split in two methods while we are combining into one.

Reply:
Android has two methods, but iOS only has one: `Amplify.Auth.signInWithWebUI(for: .facebook, presentationAnchor: self.view.window!)`